search-web/openshift/templates/bc.yaml

```diff
@@ -36,14 +36,14 @@ objects:
         ref: ${GIT_REF}
         uri: ${GIT_REPO_URL}
       dockerfile: |
-        FROM docker-remote.artifacts.developer.gov.bc.ca/node:14.15.1 as build-stage
+        FROM docker-remote.artifacts.developer.gov.bc.ca/node:14.17.6 as build-stage
         WORKDIR /app
         COPY ./package*.json ./
         RUN npm install
         COPY . .
         RUN npm run build
 
-        FROM docker-remote.artifacts.developer.gov.bc.ca/nginx:1.18.0 as production-stage
+        FROM docker-remote.artifacts.developer.gov.bc.ca/nginx:1.21.0 as production-stage
```

Comment:
you are upgrading nginx as well..is that intentional?

Reply:
That's for the vulnerability-scan testing.